index.js

```diff
@@ -602,7 +602,7 @@
   var Array$prototype$equals = function(other) {
     if (other.length !== this.length) return false;
     for (var idx = 0; idx < this.length; idx += 1) {
-      if (!equals(other[idx], this[idx])) return false;
+      if (!equals(this[idx], other[idx])) return false;
```

Comment:
In order to compare `[1, <Circular>]` and `[1, [1, <Circular>]]` we expect to compare `<Circular>` and `[1, <Circular>]` (after comparing `1` and `1`), but with the previous argument order the values would be flipped: `[1, <Circular>]` and `<Circular>`. The change above isn't necessary, but avoids confusion when debugging.